src/data/html-quiz.ts

```diff
@@ -1525,5 +1525,15 @@ const htmlQuiz = [
       "The <i> tag is used to make text italic without adding extra emphasis. It is commonly used for text that is set off from normal prose, like foreign words or thoughts. ",
     Link: "https://devdocs.io/html/element/i",
   },
+  {
+  Question:
+  "In HTML, responsive text size can be set with a vw unit, which means the",
+   Answer: "<viewport width>",
+   Distractor1: "<versatile width>",
+   Distractor2: "<viewing wide>",
+   Distractor3: "<view length>",
+   Explanation: "The full form of VW is viewport width. It works like the percentage unit. Specifying 10vw is equivalent to occupying 10% of entire visible screen width.",
+   Link: "https://www.freecodecamp.org/news/learn-css-units-em-rem-vh-vw-with-code-examples/#why-learn-css-relative-units"
```

Comment:
We already have a quiz question for viewport width in the CSS file. 
Please come up with another question.